app/models/miq_event_definition_set.rb

```diff
@@ -3,8 +3,7 @@ class MiqEventDefinitionSet < ApplicationRecord
 
   def self.seed
     existing = all.group_by(&:name)
-    CSV.foreach(fixture_path, :headers => true, :skip_lines => /^#/) do |csv_row|
-      set = csv_row.to_hash
+    YAML.load_file(fixture_path).each do |set|
```

Comment:
should this become a class method `set_definitions_from_path`?

Reply:
yeah, following the pattern in the other file, you could then `mock` / `stub` /`allow` / `expect` this method with whatever path and return a test array of hashes.